docs/UsersGuide.asciidoc

```diff
@@ -554,6 +554,19 @@ Steps 4. to 6. can be repeated for further +assert_screen+s/needles without rest
 test.
 
 
+=== Job group editor gh#2111 ===
```

Comment:
please make that a complete link with `https://github.com/os-autoinst/openQA/pull/2111[gh#2111]`